modelforge/train/training.py

```diff
@@ -665,7 +663,9 @@ def __init__(
             else dataset_statistic
         )
         self.experiment_logger = self.setup_logger()
-        self.model = self.setup_potential(potential_seed)
+        self.potential_training_adapter = self.setup_potential_training_adapter(
```

Comment:
`potential_training_adapter` is a descriptive name, but I am not sure if it is a useful name. It is long and it requires an understanding of the class structure of the training module.

maybe it makes more sense to call this lightning_module?

Reply:
This completely make sense to me!